incubator/cloudwatch-exporter/values.yaml

```diff
@@ -0,0 +1,78 @@
+# Default values for cloudwatch-exporter.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: prom/cloudwatch-exporter
+  tag: latest
```

Comment:
Can you specify a release version?

Reply:
there's no other version for this project

Reply:
Hm, that's bad.